libopae/plugins/xfpga/enum.c

```diff
@@ -384,11 +384,14 @@ STATIC fpga_result enum_fme(const char *sysfspath, const char *name,
 	}
 
 	// Discover the socket id from the FME's sysfs entry.
-	resval = sysfs_parse_attribute64(sysfspath, FPGA_SYSFS_SOCKET_ID, &value);
-	if (resval != 0) {
-		return FPGA_NOT_FOUND;
+	if (sysfs_path_isvalid(sysfspath, FPGA_SYSFS_SOCKET_ID) == FPGA_OK) {
```

Comment:
Probably want to name this sysfs_path_is_valid for consistency.  Looks ok, otherwise.

Reply:
fixed